tests/e2e/specs/merchant/checkout-terms.test.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * External dependencies
+ */
+import {
+	merchant,
+	openDocumentSettingsSidebar,
+	setCheckbox,
+	unsetCheckbox,
+} from '@woocommerce/e2e-utils';
+import {
+	visitBlockPage,
+	selectBlockByName,
+	saveOrPublish,
+} from '@woocommerce/blocks-test-utils';
+
+/**
+ * Internal dependencies
+ */
+import {
+	shopper,
+	preventCompatibilityNotice,
+	reactivateCompatibilityNotice,
+} from '../../../utils';
+import {
+	SHIPPING_DETAILS,
+	SIMPLE_VIRTUAL_PRODUCT_NAME,
+} from '../../../utils/constants';
+
+if ( process.env.WOOCOMMERCE_BLOCKS_PHASE < 2 ) {
+	// eslint-disable-next-line jest/no-focused-tests
+	test.only( `Skipping checkout tests`, () => {} );
+}
+
+describe( 'Merchant → Checkout → Can adjust T&S and Privacy Policy options', () => {
+	beforeAll( async () => {
+		await shopper.goToShop();
+		await page.goto( 'http://localhost:8889/?setup_terms_and_privacy' );
```

Comment:
Instead of `http://localhost:8889/`, we can use `BASE_URL`. We'd be able to avoid using hardcoded values this way.